doc/classes/JavaClassWrapper.xml

```diff
@@ -18,6 +18,7 @@
 		[b]Warning:[/b] When calling Java methods, be sure to check [method JavaClassWrapper.get_exception] to check if the method threw an exception.
 	</description>
 	<tutorials>
+		<link title="Integrating with Android APIs">$DOCS_URL/tutorials/platform/android/javaclasswrapper_and_androidruntimeplugin.html</link>
```

Comment:
This PR was merged too early before the accompanying docs PR, so this link is broken (and breaks CI on godot-docs)